priam/src/main/java/com/netflix/priam/config/IConfiguration.java

```diff
@@ -403,6 +398,10 @@ default String getRestoreSnapshot() {
         return StringUtils.EMPTY;
     }
 
+    default String getRestoreDataLocation() {
+        return getCassandraBaseDirectory() + "/restore";
```

Comment:
To confirm my understanding on our systems this would be something like `/mnt/data/cassandra/restore`? If so, +1